Intellij IDEA Tutorial.md

```diff
@@ -0,0 +1,95 @@
+# IntelliJ IDEA Tutorial
+
+## **1. Download IntelliJ IDEA Community Edition**
```

Comment:
Isn’t it enough for this to be a title? Why do we need to put it in bold as well?

Reply:
I made a commit that changed it.